cmd/openshift-router/main.go

```diff
@@ -10,16 +10,14 @@ import (
 
 	"github.com/spf13/cobra"
 
-	"k8s.io/apiserver/pkg/util/logs"
 	"k8s.io/client-go/pkg/version"
 
 	"github.com/openshift/library-go/pkg/serviceability"
+
 	"github.com/openshift/router/pkg/cmd/infra/router"
 )
 
 func main() {
-	logs.InitLogs()
```

Comment:
How come we don't need this (where "this" is logging) anymore?

Reply:
Good point—should we also be switching glog out for klog?

Reply:
I found the moved package an restored it. https://github.com/kubernetes/kubernetes/commit/ba81a5409affb6b861d5994c1e2f7c74a05826e5

I don't want to re-do log handling in this PR unless absolutely necessary

Reply:
> Good point—should we also be switching glog out for klog?

I would say so. Looks like everything that is in vendor is using klog.

Reply:
> I don't want to re-do log handling in this PR unless absolutely necessary

Agreed. Should be a separate PR.